### ISSUE TITLE
Reorganization

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-pub#29]
I see the following problems with this repo:

- Its name is too short and is easily confused with [ACLPUB](https://github.com/acl-org/ACLPUB)
- It is weirdly reported as a fork of naacl-org version, which makes it seem unofficial
- It is oddly on the `gh-pages` branch.

I would like to propose doing the following:

- Renaming it to "acl-conference-publications"
- Disassociating it from the naacl-org repository
- Switching to a Github-pages repo that operates on the "master" branch.

Comments, objections, etc?

CC: @acl-org-github @slukin @davidweichiang @danielgildea

**Comment by danielgildea** [comment]
> - Its name is too short and is easily confused with [ACLPUB](https://github.com/acl-org/ACLPUB)

> I would like to propose doing the following:
> 
> - Renaming it to "acl-conference-publications"

The two names are still very similar.  Maybe
"acl-conference-documentation"
or
"acl-chair-info"
or something?

Dan

**Comment by mjpost** [comment]
I like these better, though maybe they are too general (e.g., chair could be program chair, etc)? What about aclpub-documentation or aclpub-info?

**Comment by mjpost** [comment]
Or even "pubdocs".

When all's done we'll advertise the URL (per #23), e.g.,: https://acl-org.github.io/pubdocs/

**Comment by davidweichiang** [comment]
I agree with @mjpost; there's already a [conference handbook](https://aclweb.org/adminwiki/index.php?title=Conference_Handbook) and this repo can't look like it duplicates it.

There is also a little bit of publications-related information in the ACL WIki:
- https://aclweb.org/adminwiki/index.php?title=Publications_chair_duties
- https://aclweb.org/adminwiki/index.php?title=Overview_of_repository_organization
- https://aclweb.org/adminwiki/index.php?title=Instructions_on_how_to_access_the_repository

I think there should be as little overlap as possible between this repo and those pages on the Wiki as well.

**Comment by mjpost** [comment]
We should clean up the ACL wiki as part of this. I have done this here and there piecemeal.

I suggest we keep the official documentation here in repos, both because I'd rather work on Github than inside the ACL wiki, which has the feel of a graveyard, and on the principle that the documentation should be kept as close to the code (data, templates, etc) as possible.

**Comment by desilinguist** [comment]
Please note that wikis are going to disappear either this year or next year as the relevant information is incorporated into the [new website](https://aclweb.netlify.app) that's under development.

**Comment by danielgildea** [comment]
On Tue, May 12, 2020 at 10:32AM, Matt Post wrote:
> Or even "pubdocs".

I like it.

**Comment by logological** [comment]
> I would like to propose doing the following:
> 
>     * Renaming it to "acl-conference-publications"
> 
>     * Disassociating it from the naacl-org repository

I think these two are very sensible.

>     * Switching to a Github-pages repo that operates on the "master" branch.

Could you explain a bit more what this means and what the advantages are?

**Comment by mjpost** [comment]
> Could you explain a bit more what this means and what the advantages are?

It's purely aesthetic: the purpose of this repo is documentation; why use any branch other than `master`?

(As for [the `gh-pages` branch in general](https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source), it has always felt to me like an ugly documentation hack that goes against the purpose of branches. Branches are meant to track the state of experimental content that has a shared history with that of the `master` branch and will one day be merged into it or dropped. Adding a `gh-pages` branch with content that is entirely divergent, and has no shared history with the main branch, is ugly. Much better to use a `docs/` subdirectory, at least.)

**Comment by logological** [comment]
Thanks, but what I was actually asking about was what a "Github-pages repo" is and why we'd want to use it, not about what branch to use in the Git repository.  Your link (via a couple hops) goes some way to explaining what GitHub Pages is (i.e., a web hosting service that serves static web pages from a Git repository).  But I wonder whether it makes sense from a content separation perspective for us to host the paper style files in a repository that is intended to host a website.  Perhaps we should split this repo into separate ones, one for the publication procedure documents, and another for the actual LaTeX/BibTeX styles (and associated documentation)?  We could always make the latter a Git submodule of the former, but keeping the latter separate I think would be cleaner (and also allow publication chairs and other interested parties to fork the style files for their own purposes without pulling in a lot of irrelevant material).

**Comment by bethard** [comment]
I would like to see us move most of the associated documentation for the LaTeX/BibTeX styles to a website. Having it in only the sample `.tex` that everyone downloads and immediately overwrites means that most people don't have an easy reference anywhere for the actual formatting instructions. I think we should slim down the LaTeX/BibTeX style files to the absolute minimum needed to provide an example, and move all the descriptive part of those style files to the web, say `submission-formatting.md`. Since we would be hosting these style instructions in the same way we already do for `camera-ready-faq.md`, I would argue against two repositories.

**Comment by mjpost** [comment]
I'm picking this back up, I hope in time for the next conference season.

Our previous decision was to rename this as "pubdocs" and consolidate @alexanderkoller's https://github.com/aclpub/aclpub. But it seems to me one option we haven't discussed is rolling this into ACLPUB. This seems like a natural place for this information (including the template files). Are there any objections or downsides I'm not considering?

**Comment by alexanderkoller** [comment]
One reason why we decided to put the information on Github is so it could be easily updated by future pub chairs. How easy is it to change documentation that is part of ACLPUB?

**Comment by mjpost** [comment]
[ACLPUB](https://github.com/acl-org/ACLPUB) is also on Github. We could merge in other documentation and also make a web page that would live at https://acl-org.github.io/ACLPUB/ (there's already something there, so we'd be updating it).

**Comment by mjpost** [comment]
Hi @alexanderkoller, others,

We have merged these sister repos (CC #23) [here](https://acl-org.github.io/ACLPUB/). This retains much of Alexander's documentation and docs from other places and can be considered a first draft that we'll continue to update. Comments are welcome over there.

matt

**Comment by davidweichiang** [comment]
I'd like to see the final-version.md merged in or made more complementary to formatting.md. Right now the table of contents gives the impression that formatting.md is only about review versions.

I can take a look at this, but next week.

**Comment by mjpost** [comment]
This makes sense. I merged these, got rid of `final-version.md`, and added a file containing START-specific notes. [See what you think](https://acl-org.github.io/ACLPUB/formatting.html).

**Comment by davidweichiang** [comment]
Uh oh, I already made PR https://github.com/acl-org/ACLPUB/pull/43 and was in the middle of further merging formatting.md and final-version.md.
